src/Http/Controllers/Family.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Choredo\Http\Controllers;
+
+use Choredo\Entities;
+use League\Fractal\Manager;
+use League\Fractal\Resource\Item;
+use Ramsey\Uuid\Uuid;
+use Ramsey\Uuid\UuidInterface;
+
+class Family extends Controller
```

Comment:
I've preferred 1:1 mapping between endpoints and controllers in the past, which allows us to make Controllers invokable. 

```
class ShowFamily
{
    public function __invoke(ServerRequestInterface $request, ResponseInterface $response, array $vars) : ResponseInterface
    {

    }
}
```

What do you think of that?

Reply:
That's my preference as well, I was following from the lumen docs.